libethcore/TransactionBase.cpp

```diff
@@ -46,28 +46,27 @@ TransactionBase::TransactionBase(TransactionSkeleton const& _ts, Secret const& _
 
 TransactionBase::TransactionBase(bytesConstRef _rlpData, CheckTransaction _checkSig)
 {
-	int field = 0;
-	RLP rlp(_rlpData);
+	RLP const rlp(_rlpData);
```

Comment:
`RLP` class is immutable, so usually I don't bother declaring it const

Reply:
I see.  I didn't know that.